### DISCUSSION TITLE
how to export and import data into different layer

### DISCUSSION THREAD

**Discussion by anwar0908** [prisma/prisma#15317] [Q&A] [answered]
Hi,
I am having issue's while importing data into different layer, actually the thing is that i have created service's(DB SERVICES, CONTROLLER SERVICES, other-services...) where i have fetched data into db-services from the prisma.ModelName using the prisma-client and exported the same data as module but when i am trying to access the same in the controller layer i am unable to do that. I am able to access the method but unable to access the actual data.

If someone could help me with that..

I am using Node js v16.15.0 with express and mysql with prisma

**Comment by anwar0908** [comment]
`in db services`

import { PrismaClient } from "@prisma/client";
const prisma = new PrismaClient();

class UserDatabase {

    async getAll(req, res) {
        try {

            const user = await prisma.AppUsers.findMany()
            // console.log(user)
            res.status(200).send( JSON.stringify(user, (_, v) => typeof v === 'bigint' ? `${v}n` : v)
            .replace(/"(-?\d+)n"/g, (_, a) => a))
            
        } catch (err) {
            // res.send({ status: false, msg: err.message });
        }
            
    }

}

export default new UserDatabase();





in controller services

import  db  from './user.db.service.js'

console.log(db)

class Userservice {

    
    getAll(){
       return new Promise((resolve,reject)=>{
        db.getAll()
        .then((res)=>{
            resolve(res)
        }).catch((error)=>{
            reject(error)
        })
       })
    }

}
export default new Userservice();

**Comment by nurul3101** [comment, marked as answer]
Hey @anwar0908 👋 
What do you get when you log the import from db service?
It's unclear what you are trying to achieve, can you please elaborate?

**Reply by anwar0908** [reply]
Hey @nurul3101 
thanks for the reply, when i try to log the db services i was not getting anything at that time. and it got resolved the issue was with the database. i was trying that with different tables but it was just the same but luckily when i used a different db it got solved.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
